modules/cookbook/pages/recipe-onboarding.adoc

```diff
@@ -0,0 +1,122 @@
+[[cookbook-recipe-onboarding]]
+= Recipe: Onboarding
+
+{productname} is all about managing client systems.
+So one of the first things you need to do is onboard some clients.
+This recipe shows you how to set up your {productname} Server to manage a new client, set up the software channels you need, and bootstrap the client using an activation key.
+
+
+
+== Problem
+
+This recipe shows you how to onboard a client to your {productname} Server.
+
+The client can be running any Linux operating system.
+For a list of supported client systems, see xref:installation:client-requirements.adoc[].
+
+This is one of the first tasks you need to do when you set up {productname} for the first time, and you will probably have to do it many more times as you use the product.
+
+This recipe shows you how to onboard a single client, for a recipe about onboarding multiple clients at the same time, see xref:FIXME[].
```

Comment:
```suggestion
```
This xref is distracting.  If needed we can add it at the end of this chapter, imo.